images/bootstrap/Dockerfile

```diff
@@ -90,6 +90,13 @@ RUN if test "${ARCH}" != s390x; then \
         rm -rf /root/.cache/bazel; \
     fi
 
+RUN if test "${ARCH}" == s390x; then \
```

Comment:
Please leave a comment why we can't use the above run with gcloud

Reply:
Thanks for the comment. Added the inline comment and updated commit message to convey that gcloud based installation of kubectl isn't done in s390x, as gcloud cli isn't supported on s390x.